### ISSUE TITLE
[typescript-fetch] response parser

### ISSUE THREAD

**Issue by farcaller** [swagger-api/swagger-codegen#4785]

**Comment by wing328** [comment]
@farcaller based on my understanding, there's no TS lib that can serialize/deserialize between JSON and TS object with proper attribute mapping. 

There's an enhancement to TS Angular2 (not node) that can handle deserialization with proper attribute mapping via https://github.com/swagger-api/swagger-codegen/pull/4264

Please pull the latest 2.3.0 to generate the client to see if that meets your needs.

**Comment by zoltanszekely** [comment]
@wing328 #4264 is for `typescript-node`, not angular2

**Comment by wing328** [comment]
@zoltanszekely yes, you're right. Would you have time to port the enhancement from #4264 to TS Angular2?
